locales/tr.po

```diff
@@ -4753,23 +4759,26 @@ msgstr ""
 msgid ""
 "The load() command of Maxima versions >5.38 is able to load .wxm files as "
 "library of maxima's commands."
-msgstr ""
+msgstr "5.38 den yukarı Maxima versiyonları .wxm dosyalarını"
+"maxima komutlarının kütüphanesi olarak yükler"
 
 #: /home/gunter/src/wxmaxima/src/TextCell.cpp:188
```

Comment:
Attention:
 +msgstr "5.38 den yukarı Maxima versiyonları .wxm dosyalarını"
		+"maxima komutlarının kütüphanesi olarak yükler"
		 
Doesn't introduce a space in front of Maxima which is a thing that has completely taken me by surprise when adding the new string.